contracts/Makefile

```diff
@@ -17,4 +17,11 @@ fmt-check:
 	forge fmt --check
 
 deps:
-	yarn
\ No newline at end of file
+	yarn
+
+# brew install typos-cli
+typos:
+	typos . --exclude 'lib/**' --write-changes
+
+diff-typos:
+	typos . --exclude 'lib/**' --diff
```

Comment:
is the idea that we'll integrate this into CI eventually? if so recommend adding a todo specifying that the dep should be added to the mise toml and at some point it should be populated into contracts CI